src/sagemaker/fw_utils.py

```diff
@@ -93,6 +93,13 @@
     "pytorch-serving-eia": [1, 3, 1],
 }
 
+INFERENTIA_VERSION_RANGES = {
```

Comment:
Why do we need to introduce the unbound and is_version_equal_or_lower? DLCs only have the lowest version. should we be consistent here? Is it because Neo/Inferentia doesn't support the latest version? EIA doesn't support the latest version either. I am okay with introducing a new map here, but maybe we should move EI's validation into the new map as well? downside of this approach is we have to keep updating the unbound every time Neo/EI supports a new version. We should put comments here about "Once Neo/EI has catched up, move them to the MERGED_FRAMEWORKS_LOWEST_VERSIONS map". What do you think?

Reply:
The reason why I introduce this upper limit is that Inferentia does not support latest versions and inferentia containers does not align with DLC versions very much.

Now we have tensorflow up to 2.1, but inferentia is 1.15.0, 3 versions behind (1.15.2, 2.0, 2.1). And for mxnet, DLC skipped 1.5.0 but inferentia only has 1.5.0. Plus inferentia is neo containers and follow neo image naming schema, so I think it’s better to have its own logic.

Also inferentia is neo containers, we should not move them to MERGRD_FRAMEWORKS_LOWEAST_VERSIONS because they are not merged dlc containers.